tools/cli/go-whisk-cli/commands/action.go

```diff
@@ -822,6 +825,18 @@ func printActionGetWithField(entityName string, field string, action *whisk.Acti
     printField(action, field)
 }
 
+func printActionGetWithURL(entityName string, actionURL string) {
+    fmt.Fprintf(
+        color.Output,
+        wski18n.T(
+            "{{.ok}} got action {{.name}}, displaying url\n",
```

Comment:
i think using the same `{{.ok}} got action {{.name}}` as before is sufficient (i.e. no need for the `,displaying url`).  the action contents that follow adhere to whatever filters were specified on the command line - which is currently just `--url` at the moment, but could be expanded upon in the future.